guides/common/modules/proc_managing-and-monitoring-hosts-using-cockpit.adoc

```diff
@@ -5,14 +5,7 @@ You can access the {Cockpit} UI through the {ProjectWebUI} and use the functiona
 
 .Prerequisites
 * You have enabled {the-Cockpit} in {Project}.
-* You have installed {the-Cockpit} on the host that you want to view:
-ifndef::satellite,orcharhino[]
-** For more information, see https://cockpit-project.org/running.html[Running Cockpit].
-endif::[]
-ifdef::satellite[]
-** For {RHEL}{nbsp}8, see {RHELDocsBaseURL}8/html/managing_systems_using_the_rhel_8_web_console/getting-started-with-the-rhel-8-web-console_system-management-using-the-rhel-8-web-console#installing-the-web-console_getting-started-with-the-rhel-8-web-console[Installing the web console] in _Managing systems using the RHEL{nbsp}8 web console_.
-** For {RHEL}{nbsp}7, see {RHELDocsBaseURL}7/html/managing_systems_using_the_rhel_7_web_console/getting-started-with-the-rhel-web-console_system-management-using-the-rhel-7-web-console#installing-the-web-console_getting-started-with-the-web-console[Installing the web console] in _Managing systems using the RHEL{nbsp}7 web console_.
-endif::[]
+* You have installed {the-Cockpit} on the host that you want to view.
```

Comment:
Should this mention foreman ships with a job template to do this?

Reply:
@adamruzicka What is the template called? I can't find it.

Reply:
`Service Action - Enable Web Console`, sadly it is only there if the users have the ansible plugin enabled
https://github.com/theforeman/foreman_ansible/blob/master/app/views/foreman_ansible/job_templates/service_action_-_enable_web_console.erb

Reply:
Thanks! I found it. I think it's a good idea to let users know about the template but I'm not sure how to best do it. Let's see what other writers think about adding a note.